modular_skyrat/code/modules/ruins/lavalandruin_code/misc.dm

```diff
@@ -20,14 +20,19 @@
 	w_class = WEIGHT_CLASS_BULKY
 
 /obj/item/katana/goldbrand/Initialize()
-	..()
+	. = ..()
 	START_PROCESSING(SSobj,src)
 
+/obj/item/katana/goldbrand/Destroy()
+	. = ..()
+	STOP_PROCESSING(SSobj,src)
```

Comment:
I feel that stop_processing should probably be put before the . = ..()

Reply:
It is functionally no different, but i'll do it